src/enzyme_ad/jax/Passes/LLVMToAffineAccess.cpp

```diff
@@ -1072,8 +1109,10 @@ convertLLVMToAffineAccess(Operation *op,
     AffineAccessBuilder &aab = *accessBuilders.back();
     auto dl = dataLayoutAnalysis.getAtOrAbove(op);
     auto res = aab.build(dl, addr);
-    if (failed(res))
+    if (failed(res)) {
       accessBuilders.pop_back();
+      remainingMemOps.push_back(std::make_pair(op, addr));
+    }
```

Comment:
instead of storing this list.

Can you just do a second walk of all store/loadops (after doing the conversion to affine where legal)?